### ISSUE TITLE
mind adding some colors?

### ISSUE THREAD

**Issue by wchristian** [danielbrauer/DiscoScience#3]
From the log:

```
   8.242 Script @__DiscoScience__/data-final-fixes.lua:39: Missing colours for the following icons: {
  ["__pycoalprocessing__/graphics/icons/science-pack-1.png"] = true,
  ["__pycoalprocessing__/graphics/icons/science-pack-2.png"] = true,
  ["__pycoalprocessing__/graphics/icons/science-pack-3.png"] = true,
  ["__pyfusionenergy__/graphics/icons/production-science-pack.png"] = true,
  ["__pyhightech__/graphics/icons/high-tech-science-pack.png"] = true
}
```

**Comment by danielbrauer** [comment]
Sure, what colours should they be?

**Comment by wchristian** [comment]
Here's the actual icons, and my guesses as to fitting colors. I'm more than happy if you decide to change values. :)

`__pycoalprocessing__/graphics/icons/science-pack-1.png` `(red:1.00, green:0.12, blue:0.18)`
![image](https://user-images.githubusercontent.com/175467/56092844-bdf28f00-5ec1-11e9-93a8-11675985e4b7.png)

`__pycoalprocessing__/graphics/icons/science-pack-2.png` `(red:0.16, green:0.50, blue:0.00)`
![image](https://user-images.githubusercontent.com/175467/56092848-c3e87000-5ec1-11e9-98d0-c9bbb5369ea3.png)

`__pycoalprocessing__/graphics/icons/science-pack-3.png` `(red:0.35, green:0.64, blue:0.84)`
![image](https://user-images.githubusercontent.com/175467/56092849-c945ba80-5ec1-11e9-8f24-8c5334505017.png)

`__pyfusionenergy__/graphics/icons/production-science-pack.png` `(red:0.66, green:1.00, blue:0.33)`
![image](https://user-images.githubusercontent.com/175467/56092857-dd89b780-5ec1-11e9-845b-91214509fa0a.png)

`__pyhightech__/graphics/icons/high-tech-science-pack.png` `(red:0.91, green:0.86, blue:0.24)`
![image](https://user-images.githubusercontent.com/175467/56092860-eed2c400-5ec1-11e9-882c-6d5de399d438.png)

**Comment by danielbrauer** [comment]
Added in dfbcf6bebb1465e4eaf21b0bad5d415991bfce2d (released as 0.1.3). Thank you!

**Comment by wchristian** [comment]
Looks nice, thanks. :)
